batch7crud-roll8/src/main/java/com/lftechnology/batch7crud/constants/ValidatorConstants.java

```diff
@@ -0,0 +1,25 @@
+package com.lftechnology.batch7crud.constants;
+
+/**
+ * Created by Grishma Shrestha grishmashrestha@lftechnology.com on 1/26/16.
+ */
+public class ValidatorConstants {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1118)